src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -204,8 +204,9 @@ public ArticlePageObject clickVEEditAndPublish(String content) {
   public ArticlePageObject publish() {
     verifyVEToolBarPresent();
     VisualEditorSaveChangesDialog save = clickPublishButton();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "save" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)